packages/nimble-components/src/mapping/text/index.ts

```diff
@@ -0,0 +1,21 @@
+import { DesignSystem } from '@microsoft/fast-foundation';
+import { Mapping } from '../base';
+import { template } from '../base/template';
+
+declare global {
+    interface HTMLElementTagNameMap {
+        'nimble-mapping-text': MappingText;
+    }
+}
+
+/**
+ * Maps data values to text.
+ */
+export class MappingText extends Mapping {}
```

Comment:
Given that this class is just a thin, concrete wrapper around the `Mapping` class, it's unclear how this class participates in actually mapping a data value to a text value. Can this be explained in the documentation for the class?